contrib/completion/bash/docker

```diff
@@ -347,6 +352,16 @@ __docker_complete_log_driver_options() {
 			" -- "${cur#=}" ) )
 			return
 			;;
+		*splunk-url=*)
+			COMPREPLY=( $( compgen -W "http https" -S "://" -- "${cur#=}" ) )
```

Comment:
@albers weird, I took `gelf-address` and `syslog-address` as examples, see https://github.com/docker/docker/pull/16488/files#diff-754420541b12dbcbbea00825942a4635R326